packages/providers/src.ts/index.ts

```diff
@@ -44,11 +44,11 @@ const logger = new Logger(version);
 // Helper Functions
 
 function getDefaultProvider(network?: Networkish, options?: any): BaseProvider {
-    if (network == null) { network = "homestead"; }
+    if (network == null) { network = "mainnet"; }
+    // @TODO: Add support for IpcProvider; maybe if it ends in ".ipc"?
 
     // If passed a URL, figure out the right type of provider based on the scheme
     if (typeof(network) === "string") {
-        // @TODO: Add support for IpcProvider; maybe if it ends in ".ipc"?
 
         // Handle http and ws (and their secure variants)
         const match = network.match(/^(ws|http)s?:/i);
```

Comment:
question: I guess that we will handle this logic once we have the non-default provider?

Reply:
Yes. I was thinking about implementing the `HederaProvider` which accepts 2 URLs right before doing this.